ci/ansible/roles/pulp/tasks/main.yaml

```diff
@@ -164,16 +164,6 @@
       priority: 50
   when: pulp_build == "beta" or pulp_build == "stable"
 
-- name: Check fips state
-  shell: cat /proc/sys/crypto/fips_enabled
-  register: fips_state
-
-- name: Install yum-plugin-priorities
```

Comment:
I'm pretty sure this is still needed, as you need yum-plugin-priorities to make priorities work

Reply:
role: pulp-priority does the same. It was a redundancy.

Reply:
Ah, cool.  Then nevermind me.